crates/runner-host/src/main.rs

```diff
@@ -35,9 +36,28 @@ fn with_state<R>(f: impl FnOnce(&mut board::State) -> R) -> R {
     f(STATE.lock().unwrap().as_mut().unwrap())
 }
 
+#[derive(Parser)]
+struct Flags {
+    #[cfg(feature = "web")]
+    #[clap(flatten)]
+    web_options: WebOptions,
+}
+
+#[derive(clap::Args)]
```

Comment:
I'm surprised you don't need `#[cfg(feature = "web")]` here. Let's see what the CI says.